SU2_CFD/src/variables/CNEMOEulerVariable.cpp

```diff
@@ -73,39 +67,18 @@ CNEMOEulerVariable::CNEMOEulerVariable(su2double val_pressure,
     Tve_Freestream = config->GetTemperature_ve_FreeStream();
   }
 
-  /*--- Allocate & initialize residual vectors ---*/
-  Res_TruncError.resize(nPoint,nVar) = su2double(0.0);
-
   /*--- Size Grad_AuxVar for axiysmmetric ---*/
   if (config->GetAxisymmetric()){
     nAuxVar = 3;
     Grad_AuxVar.resize(nPoint,nAuxVar,nDim,0.0);
     AuxVar.resize(nPoint,nAuxVar) = su2double(0.0);
   }
 
-  /*--- Only for residual smoothing (multigrid) ---*/
-  for (unsigned long iMesh = 0; iMesh <= config->GetnMGLevels(); iMesh++) {
-    if (config->GetMG_CorrecSmooth(iMesh) > 0) {
-      Residual_Sum.resize(nPoint,nVar);
-      Residual_Old.resize(nPoint,nVar);
-      break;
-    }
-  }
-
-  /*--- Allocate undivided laplacian (centered) and limiter (upwind)---*/
-  if (config->GetKind_ConvNumScheme_Flow() == SPACE_CENTERED)
-    Undivided_Laplacian.resize(nPoint,nVar);
-
   /*--- Always allocate the slope limiter,
    and the auxiliar variables (check the logic - JST with 2nd order Turb model - ) ---*/
   Limiter.resize(nPoint,nVar) = su2double(0.0);
```

Comment:
@WallyMaier, this may be leftover from when NEMO reconstructed the conservatives?

Reply:
i think this was there just because we had been trying to mirror other solvers. but in NEMO this seems to not to be used actually. @WallyMaier can you confirm ?

Reply:
in any case the change to CFlowVariable seems not to harm anything

Reply:
@pcarruscag @CatarinaGarbacz This is left over from when the other solvers were mimicked a long time ago.  Before I gave up on implementing some centered schemes. This should be superfluous.

Reply:
After futher looking through, it seems the only time NEMO uses "Limiter" is as a bool.